taxi/views.py

```diff
@@ -74,6 +76,24 @@ class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     success_url = reverse_lazy("taxi:car-list")
 
 
+def car_driver_add(request: dict, pk: int):
+    car = Car.objects.get(id=pk)
+    car.drivers.add(request.user)
+    return HttpResponseRedirect(reverse_lazy(
+        "taxi:car-detail",
+        kwargs={"pk": pk}
+    ))
+
+
+def car_driver_delete(request: dict, pk: int):
+    car = Car.objects.get(id=pk)
+    car.drivers.remove(request.user)
+    return HttpResponseRedirect(
+        reverse_lazy("taxi:car-detail",
+                     kwargs={"pk": pk}
+                     ))
```

Comment:
Change your code formatting for these lines, make it the same as for lines 82-85.